README.md

```diff
@@ -2,18 +2,8 @@
 
 > 📣 Use this repository as a starter template if you're a GitHub user interested in building your own custom Blocks!
 
-**Beware!** The videos on this page cover an earlier version of Blocks; the text of this page has the latest information.
-
 The project is built with [React](https://reactjs.org/), [Typescript](https://www.typescriptlang.org/), and [Vite](https://vitejs.dev/). We'll guide you through how to use it.
```

Comment:
I know you didn't add this part, but I wonder if we want to be a bit more clear here for the uninitiated. Maybe something like:

`This project contains React components written in Typescript and bundled with Vite. Read on to learn how to build Blocks with it!`

🤷

Reply:
🎉